src/arbitraries/types.ts

```diff
@@ -20,4 +20,6 @@ export class FluentRandomGenerator {
   }
 
   initialize() { this.generator = this.builder(this.seed) }
+
+  seeded() { return this.generator === Math.random ? undefined : this.seed }
```

Comment:
Is this from this PR?

Reply:
> Is this from this PR?

no, it is not, I can remove it and create a new PR just for that